master/static/migrations/20240416153843_add-namespaces-and-quotas.tx.up.sql

```diff
@@ -0,0 +1,19 @@
+/* Contains all of the namespaces involved in namespace-workspace bindings. */
+CREATE TABLE workspace_namespace (
+    id                      integer PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    namespace_name          varchar(63),
+    cluster_name            varchar(80) NOT NULL,
+    workspace_id            integer REFERENCES workspaces (id)
+);
```

Comment:
this is redundant with a table that Saloni's creating in her PR -- choose who is going to write the SQL migration for this, or create a separate PR just for creating this table and then both edit your PRs for that

Reply:
For sure! I'll rebase onto Saloni's PR and remove this code, just didn't want to rebase before hers was done (which could've caused build issues while I was working on this).